packages/zoe/src/contractFacet/types-ambient.d.ts

```diff
@@ -237,11 +237,11 @@ type ContractStartFn<
 > = (
   zcf: ZCF<CT>,
   privateArgs: PA,
-  baggage?: MapStore<any, any> | undefined,
+  baggage: import('@agoric/vat-data').Baggage,
 ) => ContractStartFnResult<PF, CF>;
 type ContractStartFnResult<PF, CF> = {
   publicFacet: PF;
-  creatorFacet: CF;
+  creatorFacet?: CF;
```

Comment:
publicFacet and creatorFacet are equally optional.  `startInstance.js` has
```
    const {
      creatorFacet = makeEmptyCreatorFacet(),
      publicFacet = makeEmptyPublicFacet(),
      creatorInvitation: creatorInvitationP,
      handleOfferObj,
    } = await E(zcfRoot).startZcf(
      ...
    );
```
So the contract doesn't have to provide either, but if they're missing, they are replaced with an empty facet.